### PR TITLE
Created REST API for Get Reviews by Product ID

### PR DESCRIPTION
**Pull request by akashcsoni**
**New enhancements added in product review REST API**

### CODE REVIEW

File: src/Http/Controllers/V1/Shop/Catalog/ProductReviewController.php [hunk 1]
```diff
@@ -68,4 +68,41 @@ public function store(Request $request, int $productId)
...
+            }
+
+            return $query;
+        });
+
+        if (is_null($request->input('pagination')) || $request->input('pagination')) {
+            $results = $query->paginate($request->input('limit') ?? 10);
+        } else {
+            $results = $query->get();
+        }
```
Comment (devansh-webkul) [inline src/Http/Controllers/V1/Shop/Catalog/ProductReviewController.php:101]: Can we simplify this and use the parent built-in method?

File: src/Resources/lang/en/app.php [hunk 2]
```diff
@@ -469,6 +469,7 @@
                 'configurable-error' => 'Please select at least one configurable attribute.',
 
                 'reviews' => [
+                    'get-success'       => 'Get reviews by product successfully.',
```
Comment (devansh-webkul) [inline src/Resources/lang/en/app.php:472]: Please provide translations for all languages.

File: src/Routes/V1/Shop/catalog-routes.php [hunk 3]
```diff
@@ -29,6 +29,10 @@
     });
 });
 
+Route::controller(ProductReviewController::class)->prefix('product')->group(function () {
+    Route::get('/reviews', 'getProductReview');
+});
```
Comment (devansh-webkul) [inline src/Routes/V1/Shop/catalog-routes.php:34]: We already have product prefix group in top. Use that group.

**Review by devansh-webkul — CHANGES_REQUESTED** [review]

**Comment by akashcsoni** [comment]
Hello @devansh-webkul 

I've Updated the code as you suggested.
Please review and inform me if anything else is needed.

Thanks

**Comment by devansh-webkul** [comment]
Hello,

Thank you for your efforts on this pull request. However, at this time, we won't be merging it as we do not currently require this endpoint. If the need arises in the future, we will likely handle it internally.

**Pull request closed without merging**